src/main/java/com/EstimatedDeliveryDate/Config/EstimatedDeliveryDateConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.EstimatedDeliveryDate.Config;
+
+import org.springframework.context.annotation.Bean;
+
+import org.springframework.context.annotation.Configuration;
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+
+/*Configuring with credentials to access the DynamoDB*/
+
+@Configuration
+public class EstimatedDeliveryDateConfig {
+	// providing credentials and region to access the DynamoDB
+	public AmazonDynamoDB amazonDynamoDBConfig() {
+		ProfileCredentialsProvider credentialProvider = new ProfileCredentialsProvider();
+		credentialProvider.getCredentials();
```

Comment:
Is there any use of this?

Reply:
Yeah I am providing the AWS account access key and security key to access dynamodb

Reply:
I was referring to this particular line